Pull upstream

antonu17 opened the pull request.

rnovikovP committed:
PMM-3702 Bump Go version.

idexter committed:
Merge pull request #18 from percona/PMM-3702-Bump-Go-version

PMM-3702 Bump Go version.

CVirus commented:
Don't review dashboard.json yet.

rafaeljesus commented on config.yml:
```diff
@@ -1,10 +1,31 @@
-    region: us-east-1
-  - instance: rds-mysql57
-    region: us-east-1
+  - instance: auth-service-db000-live
```
is there any option for fetching all the instances w/out the need to adding to this file?

CVirus replied:
Done .. please re-review

rafaeljesus commented on entry.py:
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```
hm I am not really sure about having py and go. Why not do it in go?

rafaeljesus replied:
If the script is too long for be managed by `sh`, we usually write it in go

CVirus replied:
Simply because I don't know go.. yet :D

rafaeljesus commented:
:shipit:

Commit:
README fix.

AlekSi committed:
PMM-4010 Update Travis CI config. (#28)

AlekSi committed:
PMM-3298 Fix crash. (#31)

AlekSi committed:
PMM-4474 Add extra labels. (#32)

AlekSi committed:
PMM-5058 Update node_exporter-like metrics. (#33)

percona-csalguero committed:
PMM-4010 Updated travis.yml

- Added go_import_path

AlekSi committed:
PMM-1772 Use stable instances for tests. (#34)

AlekSi committed:
PMM-5085 Last tweaks. (#36)

artemgavrilov committed:
PMM-5192 Remove go1.12 from .travis.yml

askomorokhov committed:
PMM-4145 Add RDS disable metric collection. (#41)

ahyield committed:
Add proxy configuration to the transport

BupycHuk committed:
PMM-5723 Reviewdog checks. (#42)

* PMM-5723 Reviewdog checks.

* Update .travis.yml

* PMM-5723 Github Token for Reviewdog.

* PMM-5723 Fix build.

AlekSi committed:
Prepare a new release. (#44)

askomorokhov committed:
PMM-6120 Use go 1.14 (#45)

askomorokhov committed:
PMM-6213 Remove go 1.13 from travis. (#46)

* PMM-6213 Remove go 1.13 from travis.

* PMM-6213 Add go 1.15.x

* PMM-6213 Add go tip.

yareach committed:
PMM-6756 Added go 1.15.x. (#48)

* [PMM-6756] Added go 1.15.x.

* PMM-6756 Update

* PMM-6756 Update

* PMM-6756 Revert

* Dummy

* PMM-6756 Update golden files

Co-authored-by: Andrii Skomorokhov <andrii.skomorokhov@percona.com>
Co-authored-by: Alexey Palazhchenko <alexey.palazhchenko@gmail.com>

vlinevych committed:
Overcome the limitation of 100 dbinstances (#47)

Commit:
ci: rotate integration keys (#52)

denisok committed:
Delete .travis.yml

askomorokhov committed:
PMM-7602 Add GitHub Action (#53)

* PMM-7602 Add GitHub Action

* PMM-7602 Point actions to forked repo.

* PMM-7602 Removed travis.

* PMM-7602 Use go modules.

* PMM-7602 Fix check.

* PMM-7602 Fix mod.

* PMM-7602 Fix setup.

* PMM-7602 Fix setup.

* PMM-7602 Fix setup.

* add environment

* PMM-7602 Fix tests.

* PMM-7602 Fix codecov

* PMM-7602 defines what to use go get or go install

* PMM-7602 Improve go get/install

* PMM-7602 Improve get promu.

* PMM-7602 Use tools module for tooling.

* PMM-7602 Fix mods.

* PMM-7602 Fix path

* PMM-7602 Fix gocoverutil

Co-authored-by: Denys Kondratenko <denisok@users.noreply.github.com>

percona-csalguero committed:
PPMM-7961 Updated exporter_shared (#59)

BupycHuk committed:
PMM-7902 Remove Go 1.15 (#60)

askomorokhov committed:
Add docker multistage build (#61)

askomorokhov committed:
bring back maxInterval=60 sec (#63)

Co-authored-by: Volodymyr Linevych <vlin@hellofresh.com>
Co-authored-by: Volodymyr Linevych <4980689+vlinevych@users.noreply.github.com>

askomorokhov committed:
PMM-8207 Cleanup. (#57)

Co-authored-by: Jiri Ctvrtka <jiri.ctvrtka.sde@gmail.com>

blovett committed:
add the ReplicaLag metric (#51)

rnovikovP committed:
Update README.md (#65)

Remove mentioning upstream as we reworked the project entirely. This statement can confuse people

nikita-b committed:
PMM-6013 change default branch to main (#67)
